<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateItemDialog</class>
 <widget class="QDialog" name="UpdateItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lblItemName">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name of Item</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Quantity:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Item Image:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbQuantity">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QLabel" name="lblImage">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoadItemImage">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>170</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Load Item Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConfirmEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>470</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Confirm Edit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
